corehq/apps/app_manager/templates/app_manager/partials/modules/case_list.html

```diff
@@ -42,15 +42,15 @@ <h4 class="panel-title panel-title-nolink">
       {% if request|toggle_enabled:'CASE_LIST_TILE' %}
         <div class="row">
           <div class="col-sm-6">
-            <select data-bind="value: caseTileTemplate" class="form-control">
-              <option value="no">{% trans "Don't Use Case Tiles" %}</option>
-              <option value="yes">{% trans "Use Case Tiles" %}</option>
-            </select>
+            <label>
+              {% trans "Case Tile Template" %}
```

Comment:
If I remember correctly the translations only become available after deploying. If not see above for a maybe better text.